### DISCUSSION TITLE
How to fulfill Stripe Checkout with Supabase

### DISCUSSION THREAD

**Discussion by sarkarology** [supabase/supabase#9978] [Questions] [answered]
I created an edge function to fulfill stripe checkout and registered the edge function's url in Stripe Webhook.

But Stripe doesn't send auth headers in when calling the webhook url (which is set to my edge function's url). And the function invocation failes with 401 Unauthorized.

How are you guys handling stripe checkouts with Supabase?

**Comment by laktek** [comment, marked as answer]
You will need to deploy your edge function with the `no-verify-jwt` option. Check the README in Stripe Webhooks example for the exact steps in deploying your function: https://github.com/supabase/supabase/tree/master/examples/edge-functions/supabase/functions/stripe-webhooks

**Reply by sarkarology** [reply]
You literally saved my day!

Thank you!
